expat/configure.ac

```diff
@@ -395,6 +395,10 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
+dnl Force the use of CMAKE_SHARED_LIBRARY_PREFIX under CMake
+CMAKE_SHARED_LIBRARY_PREFIX="\${CMAKE_SHARED_LIBRARY_PREFIX}"
```

Comment:
Note this^^ value will need to be verbatim `lib` or `cyg` for the CI to be happy.